README.md

```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
 
 ###[Ch. 18 - Type inference WIP](src/test/resources/test_ready/ch_18_type_inference)
 
+---
+
+## Examples of translation projections
+
+Bellow there are all designed mappings at the current moment.
+
+#### Identifiers
+Each identifier preserves name. There is no name mangling.
```

Comment:
This is not true though? EO does not support identifiers starting with capital letter, for example, and you at least add `class_` prefix for class names. 

Also, what happens when method is overloaded for different types/number of arguments? What if field name clashes with method name in the same class? All of these are allowed by Java, but not by EO, so you at least **should** use different identifiers. If you still decide to keep the identifiers, ignoring these issues, you must acknowledge it in the documentation.

Reply:
> Also, what happens when method is overloaded for different types/number of arguments? What if field name clashes with method name in the same class? All of these are allowed by Java, but not by EO, so you at least **should** use different identifiers. If you still decide to keep the identifiers, ignoring these issues, you must acknowledge it in the documentation.

Yes, it is a problem and it should be fixed. For now we cannot overcome such problems.

> you at least add `class_` prefix for class names.

Typo. This should be mentioned